index.html

```diff
@@ -1205,7 +1205,7 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
                   <p data-lang="zh-hans">文字由左向右，行由上向下排列。</p>
                   <p data-lang="zh-hant">文字由左而右，行由上而下排列。</p>
                   <div class="note">
-                    <p data-lang="en">Traditional Chinese mainly uses vertical writing mode. When characters need to be composed horizontally, each line will contain one character and lines will be arranged from right to left, and no columns will be applied. As a special case for vertical writing mode, this can be seen in stone inscriptions and headings in newspapers and magazines, but now it has been replaced by horizontal writing mode arranged from left to right.</p>
+                    <p data-lang="en">Traditional Chinese mainly uses vertical writing mode. When characters need to be composed horizontally, each character will be arranged in a single horizontal line from right to left, and no columns will be applied. Such a layout is considered a special case for vertical writing mode, often seen on stone inscriptions and headings for newspapers and magazines in the past. Now it has largely been replaced by the horizontal writing mode arrangement from left to right.</p>
```

Comment:
Thanks for clearing up the text about one character lines. That never sounded right to me.

Shouldn't the english translation of the first sentence say, instead: "Vertical writing mode is mainly used in Traditional Chinese." ?

There are some other issues with this note that i'll raise as a separate issue, since i assume that this english text now follows the chinese pretty accurately.

Reply:
I agree with this, and will make the change accordingly.